modules/workbench/variables.tf

```diff
@@ -55,3 +55,44 @@ variable "notification_channels" {
     }
   }]
 }
+
+#
+# Egress
```

Comment:
This duplicates `modules/workbench/modules/egress_detection/variables.tf` - is that intended?

Reply:
I think we need both. `workbench's var.tf` refers to `egress_detection's var.tf`.  But we only need to set the default value in one place. Let me delete the default value from `egress_detection's var.tf` then